checkstyle/suppressions.xml

```diff
@@ -190,6 +190,8 @@
     <suppress checks="" files="com.hazelcast.client.impl.protocol.util[\\/]"/>
```

Comment:
Why all `protocol` packages are suppressed? I think it's easier to make them checkstyle compatible during development instead of making them at the end in bulk. Even you can forget to remove this suppressions and they'll never get checked until someone notices. 

Writing this comment because I saw some checkstyle issues in the code and noticed that they all suppressed.